pandora-common/src/networking/directory_client.ts

```diff
@@ -60,7 +60,7 @@ export const DirectoryAccountSettingsSchema = z.object({
 	/**
 	 * Controls whether to show the attribute icons or preview images in small preview.
 	 */
-	wardrobeSmallPreview: z.enum(['icon', 'image']).default('icon'),
+	wardrobeSmallPreview: z.enum(['icon', 'image']).default('image'),
```

Comment:
Changing it here is not enough, update `ACCOUNT_SETTINGS_DEFAULT` lower, too.

Reply:
Oh my~ Two defaults!

Thank you, fixed it~